src/endpoints/getEvent.ts

```diff
@@ -39,13 +40,41 @@ export const getEvent = (config: HLTVConfig) => async ({
         .trim() || undefined
   }))
 
+  const relatedEvents = toArray($('.related-event')).map(eventEl => ({
+    name: eventEl.find('.event-name').text(),
+    id: Number(
+      eventEl
+        .find('a')
+        .attr('href')
+        .split('/')[2]
+    )
+  }))
+
+  function findEventByName(eventName, relatedEvents): Event | undefined {
```

Comment:
Please include the types for `eventName` and `relatedEvents`. Also the implementation would be much cleaner with just a `const event = relatedEvents.find(event => event.name === eventName)` instead of a for..of loop

Reply:
Would be good if there was a npm script ts linter that would highlight these cases. Could be even then added to a pre-commit hook.

Reply:
Yeah, I haven't spent much time making it easier for people to contribute, I'll take a note.